Every bline in this game has the player's fielding position in the sequence
column (Bassett's is NA). The sequence column is rewritten with each row's
order within its batting slot.

--- a/ngl_b/1938.EBR
+++ b/ngl_b/1938.EBR
@@ -43,25 +43,25 @@
 start,hannl102,"Leo Hannibal",1,9,1
-stat,bline,bensg101,0,1,8,4,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,willc106,0,2,6,3,0,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,bassp101,0,2,NA,1,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,wilsd103,0,3,9,3,1,1,0,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
-stat,bline,hairn101,0,4,7,3,0,0,0,0,0,-1,1,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,willh105,0,5,4,5,0,1,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,washj102,0,6,3,4,0,1,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,clarb101,0,7,5,0,0,0,0,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
-stat,bline,banks101,0,7,5,1,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,ruffl101,0,8,2,4,1,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,bensg101,0,1,1,4,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,willc106,0,2,1,3,0,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,bassp101,0,2,2,1,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,wilsd103,0,3,1,3,1,1,0,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
+stat,bline,hairn101,0,4,1,3,0,0,0,0,0,-1,1,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,willh105,0,5,1,5,0,1,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,washj102,0,6,1,4,0,1,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,clarb101,0,7,1,0,0,0,0,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
+stat,bline,banks101,0,7,2,1,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,ruffl101,0,8,1,4,1,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
 stat,bline,taylj108,0,9,1,2,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,cainm101,0,9,1,2,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,smitt105,0,9,1,0,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,benjj101,1,1,8,4,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,carlm101,1,2,4,5,2,2,0,1,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,harrv102,1,3,7,4,2,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,gibsj101,1,4,2,3,2,2,2,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
-stat,bline,leonb101,1,5,3,4,2,3,1,1,0,-1,0,-1,-1,1,-1,-1,1,-1,-1,-1
-stat,bline,willj110,1,6,9,4,0,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,johnj112,1,7,5,4,0,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
-stat,bline,jackj104,1,8,6,3,0,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,cainm101,0,9,2,2,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,smitt105,0,9,3,0,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,benjj101,1,1,1,4,1,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,carlm101,1,2,1,5,2,2,0,1,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,harrv102,1,3,1,4,2,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,gibsj101,1,4,1,3,2,2,2,0,0,-1,0,-1,-1,2,-1,-1,0,-1,-1,-1
+stat,bline,leonb101,1,5,1,4,2,3,1,1,0,-1,0,-1,-1,1,-1,-1,1,-1,-1,-1
+stat,bline,willj110,1,6,1,4,0,0,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,johnj112,1,7,1,4,0,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
+stat,bline,jackj104,1,8,1,3,0,1,0,0,0,-1,0,-1,-1,1,-1,-1,0,-1,-1,-1
 stat,bline,hannl102,1,9,1,1,0,0,0,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
-stat,bline,welmr101,1,9,1,3,1,2,1,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
+stat,bline,welmr101,1,9,2,3,1,2,1,0,0,-1,0,-1,-1,0,-1,-1,0,-1,-1,-1
 stat,pline,taylj108,0,1,15,-1,26,8,-1,-1,0,6,-1,4,-1,2,-1,-1,-1,0,-1
